src/components/Sam.tsx

```diff
@@ -1,7 +1,156 @@
+// import React from 'react';
+
+// const Sam = () => {
+// return (<div>Sam</div>);
+// };
+
+// export default Sam;
 import React from 'react';
+import { Theme, createStyles, makeStyles } from '@material-ui/core/styles';
+import ExpansionPanel from '@material-ui/core/ExpansionPanel';
+import ExpansionPanelDetails from '@material-ui/core/ExpansionPanelDetails';
+import ExpansionPanelSummary from '@material-ui/core/ExpansionPanelSummary';
+import Typography from '@material-ui/core/Typography';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+
+// List Items
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import Divider from '@material-ui/core/Divider';
+
+// Form Input
+import Card from '@material-ui/core/Card';
+import CardActions from '@material-ui/core/CardActions';
+// import CardContent from '@material-ui/core/CardContent';
+import TextField from '@material-ui/core/TextField';
+import Grid from '@material-ui/core/Grid';
+import Box from '@material-ui/core/Box';
+
+const useStyles = makeStyles((theme: Theme) => createStyles({
+  root: {
+    width: '25%'
+  },
+  heading: {
+    fontSize: theme.typography.pxToRem(15)
+  },
+  secondaryHeading: {
+    fontSize: theme.typography.pxToRem(15),
+    color: theme.palette.text.secondary
+  },
+  icon: {
+    verticalAlign: 'bottom',
+    height: 20,
+    width: 20
+  },
+  details: {
+    alignItems: 'center'
+  },
+  column: {
+    flexBasis: '33.33%'
+  },
+  helper: {
+    borderLeft: `2px solid ${
+      theme.palette.divider
+      }`,
+    padding: theme.spacing(1, 2)
+  },
+  link: {
+    color: theme.palette.primary.main,
+    textDecoration: 'none',
+    '&:hover': {
+      textDecoration: 'underline'
+    }
+  },
+  listItem: {
+    width: '100%',
+    maxWidth: 360,
+    backgroundColor: theme.palette.background.paper
+  },
+  inputField: {
+    width: '30ch',
+  },
+  spacedBox: {
+    float: 'right',
+  },
+}));
 
-const Sam = () => {
-  return (<div>Sam</div>);
-};
+export default function DetailedExpansionPanel() {
+  const classes = useStyles();
 
-export default Sam;
+  return (
+    <Grid container spacing={3}>
+      <div className={
+        classes.root
+      }>
+        <Box paddingTop={2} p={3}>
```

Comment:
I think it's along this line, that we have a bit of a hiccup for the padding. The navbar is pretty much covered except for a silver of the top of these links, from the padding; maybe switching to margins could help